### DISCUSSION TITLE
Ability to measure the weight of metrics created in Power BI

### DISCUSSION THREAD

**Discussion by Glovecm16** [sql-bi/Bravo#424] [Q&A] [answered]
Hello everybody,

I would like to know if with Bravo it is possible to analyze the memory consumed by the metrics created with DAX within Power BI.

Thanks :)

**Comment by marcosqlbi** [comment, marked as answer]
It is not possible to measure the memory consumed by a DAX measure, you can measure the cost of a query execution using the Server Timings pane in DAX Studio.

**Reply by Glovecm16** [reply]
Thanks a lot for the reply!
